moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -1,197 +1,67 @@
-                  <EditNoteOutlinedIcon />
-                </IconButton>
-              </Tooltip>
-
-              <Tooltip
-                title={
-                  isSignal
-                    ? "Signal locations can only be changed by editing the component details"
-                    : "Map"
-                }
-              >
-                {/* this span allows the tooltip to display when IconButton is disabled */}
-                <span>
-                  <IconButton
-                    color="primary"
-                    aria-label="map"
-                    onClick={onEditMap}
-                    disabled={isSignal}
-                  >
-                    <EditLocationAltOutlinedIcon
-                      color={
-                        getIsComponentMapped(component) ? undefined : "error"
-                      }
-                    />
-                  </IconButton>
-                </span>
-              </Tooltip>
-
-              <Tooltip
-                PopperProps={{
-                  disablePortal: true,
-                }}
-                open={!!copiedUrl}
-                disableFocusListener
-                disableHoverListener
-                disableTouchListener
-                title="Copied!"
-                placement="top"
-              >
-                {/* This span prevents warning about providing title prop to child of Tooltip */}
-                <span>
-                  <Tooltip title="Copy link to component" placement="bottom">
-                    <IconButton
-                      color="primary"
-                      aria-label="link"
-                      onClick={copyLinkToClipboard}
-                    >
-                      <LinkIcon />
-                    </IconButton>
-                  </Tooltip>
-                </span>
-              </Tooltip>
+    <List sx={{ paddingTop: 0 }}>
+      {allComponents.map((component) => {
+        const isProjectComponent = component.project_id === parseInt(projectId);
+        const isExpanded = getIsExpanded(component);
 
-              <Tooltip title="Move to another project">
-                <IconButton
-                  color="primary"
-                  aria-label="move"
-                  onClick={() => {
-                    setIsMovingComponent(true);
-                  }}
-                >
-                  <DriveFileMoveOutlinedIcon />
-                </IconButton>
-              </Tooltip>
-
-              <Tooltip title="Delete">
-                <IconButton
-                  color="primary"
-                  aria-label="delete"
-                  onClick={() => {
-                    setIsDeletingComponent(true);
-                  }}
-                >
-                  <DeleteOutlinedIcon />
-                </IconButton>
-              </Tooltip>
-            </Stack>
-          }
-        />
-      );
-    })
+        if (isProjectComponent) {
+          return (
+            <ProjectComponentListItem
```

Comment:
suchh a tiny thing but could we make the name of the file the same as the name of the component? The file is `ProjectComponentsListItem`

Reply:
Thanks for having a sharp eye and catching this. 🙏 🙏 Just pushed this change